README.md

```diff
@@ -2,7 +2,7 @@
 
 *Light container updating DNS A and/or AAAA records periodically for multiple DNS providers*
 
-[![DDNS Updater by Quentin McGaw](https://github.com/qdm12/ddns-updater/raw/master/readme/title.png)](https://hub.docker.com/r/qmcgaw/ddns-updater)
+<a href="https://hub.docker.com/r/qmcgaw/ddns-updater"><img src="https://github.com/qdm12/ddns-updater/raw/master/readme/ddnsgopher.svg" width="200" /></a>

```

Comment:
Minor HTML-ish change

```suggestion
<img height="200" src="https://raw.githubusercontent.com/qdm12/ddns-updater/master/ddnsgopher.svg?sanitize=true">
```

Reply:
I didn't commit this change because the URL was missing the /readme. However, I implemented the suggested change. I didn't know about the sanitize option ^^ also I agree that limiting the height is more important than limiting the width. I also added an attribute `alt` to the image tag for accessibility

Reply:
Oh nice you put in the readme directory 👍  I was actually thinking to myself we should move it to a readme directory yesterday 😄